docs/design/epm/design.md

```diff
@@ -0,0 +1,206 @@
+# Overview
+
+This document introduces EPM fundamental and design. EPM is a module optimized enclave boot time. It is designed as two parts, one is as OCI bundles cache management, the other one is as enclave cache pool management. This documentation will focus on enclave cache pool management currently, briefly call it as EPM as below. 
```

Comment:
change `one is as OCI bundles cache management, the other one is as enclave cache pool management` to `one is OCI bundles cache management, the other one is enclave cache pool management`